packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -775,9 +956,12 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
   @override
   String get displayName => name;
 
+  final bool isTag;
```

Comment:
can you add some dart doc here to describe that when this is true, this is a synthetic stack frame that is not actually part of the call stack tree